receptorctl/receptorctl/cli.py

```diff
@@ -28,6 +27,17 @@ def parse_args(self, ctx, args):
                 param.required = False
             return super(IgnoreRequiredWithHelp, self).parse_args(ctx, args)
 
+def pprint(json_data):
+    click.echo(json.dumps(json_data, indent=4, sort_keys=True))
+
+def printWarning(message):
+    click.echo(click.style(f"Warning: {message}", fg='magenta'), err=True)
+
+def printError(message, exitCode=None):
+    click.echo(click.style(f"ERROR: {message}", fg='red'), err=True)
+    if exitCode:
```

Comment:
I don't like the idea of exiting the program as a side effect of a print function, it makes the code difficult to follow. I had to go through the rest of your diff and find when you were passing `1` as an (unnamed) argument to this function.

Reply:
I probably should have made a print_error_and_exit function to be more explicit but went back to just a print_error followed by sys.exit.